massa-node/base_config/initial_vesting.json

```diff
@@ -0,0 +1,46 @@
+{
+    "AU1LQrXPJ3DVL8SFRqACk31E9MVxBcmCATFiRdpEmgztGxWAx48D": [
+        {
+            "slot": {
+                "period": 0,
+                "thread": 0
+            },
+            "min_balance": "100000",
+            "max_rolls": 45
+        },
+        {
+            "slot": {
+                "period": 4,
+                "thread": 0
+            },
+            "min_balance": "50000",
+            "max_rolls": 50
+        },
+        {
+            "slot": {
+                "period": 40,
+                "thread": 0
+            },
+            "min_balance": "0",
+            "max_rolls": 18446744073709551615
+        }
+    ],
+    "AU1DHJY6zd6oKJPos8gQ6KYqmsTR669wes4ZhttLD9gE7PYUF3Rs": [
+        {
+            "slot": {
+                "period": 35,
+                "thread": 4
+            },
+            "min_balance": "100000",
+            "max_rolls": 10
+        },
+        {
+            "slot": {
+                "period": 30,
+                "thread": 6
+            },
+            "min_balance": "0",
+            "max_rolls": 18446744073709551615
+        }
+    ]
```

Comment:
Can you increase the periods number of each vesting range so that we have few days on each (a period is 16 secondes) ? And also add a address with coins vested until 1 milion period without any roll authorized so that we have a usecase that is similar to decentralization pool ?

Reply:
I set in vesting file : 
First address range is 0 - 5 - 10 Days
The second is 0 - 5 days
The third is 0 - 185 days (1_000_000 period)